src/absil/il.fs

```diff
@@ -8,14 +8,18 @@ module (*internal*) Microsoft.FSharp.Compiler.AbstractIL.IL
 
 
 open Internal.Utilities
+#if FABLE_COMPILER
```

Comment:
It's not that there is no implicit opening, it's the order of opening after the adapters are loaded which overwrite some core types.